src/main/java/team9/ddang/member/repository/MemberRepository.java

```diff
@@ -18,4 +17,12 @@ public interface MemberRepository extends JpaRepository<Member, Long> {
 
     @Query("SELECT m FROM Member m WHERE m.family.familyId = :familyId AND m.isDeleted = 'FALSE'")
     List<Member> findAllByFamilyId(@Param("familyId") Long familyId);
+
+    @Query("""
+    SELECT m 
+    FROM Member m 
+    WHERE m.family.familyId =
+    (SELECT m2.family.familyId FROM Member m2 WHERE m2.memberId = :memberId)
+""")
```

Comment:
저의 얕은 지식이긴 한데 서브쿼리를 EXISTS로 사용하면 성능상 다소 이점이 있다고 알고 있습니다. 수정사항은 아니고 간단한 참고만 부탁드립니다!

```suggestion

    @Query("""
    SELECT m
    FROM Member m
    WHERE EXISTS (
        SELECT 1
        FROM Member m2
        WHERE m2.memberId = :memberId
        AND m.family.familyId = m2.family.familyId
    )
```

Reply:
아하 그렇군요! 추후 리팩토링 시 참고해서 하겠습니다!